packages/routes/src/RouteContext.ts

```diff
@@ -11,10 +13,25 @@ import * as rfs from 'require-from-string'
 export class RouteContext {
   request: Request
   response: Response
+  injector = new Container()
+  diModule: ContainerModule | undefined
 
-  constructor(request: Request, response: Response) {
+  constructor(
+    request: Request,
+    response: Response,
+    diModule?: ContainerModule
```

Comment:
I dont see you attaching the Container to a RouteContext from a handling Route. Is this needed?
I would like you to expand a helloworld example to show us how this should be used.
From your other PR i dont see how the diModule is passed from Route to RouteContext. Maybe you confuse the `this` context of the handler Method. It is not bound to Route so this wont give you the diModule...

Reply:
Maybe also write a few tests showing that this still works with RouteTest.invokeMocked
I rely on the behavior to exchange the concrete RouteContext class behind the Scences. We should change this alltogether but we cant right now...

Reply:
True, true. Changed the binding logic to run within the route handling pipeline. Added some tests as well, on both PRs.